src/Symfony/Component/VarDumper/Tests/Caster/GmpCasterTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Tests\Caster;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\VarDumper\Caster\GmpCaster;
+use Symfony\Component\VarDumper\Cloner\Stub;
+use Symfony\Component\VarDumper\Test\VarDumperTestTrait;
+
+class GmpCasterTest extends TestCase
+{
+    use VarDumperTestTrait;
+
+    /**
+     * @requires extension gmp
+     */
+    public function testCastGmp()
+    {
+        $gmpString = gmp_init('1234');
+        $gmpOctal = gmp_init(010);
+        $gmp = gmp_init('01101');
```

Comment:
we also need a few assertions that just checks the dump of these dumper directly, which going through the caster (well, doing it indirectly).